source/acquaman/VESPERS/VESPERSScanController.cpp

```diff
@@ -46,22 +46,7 @@ AMAction3 *VESPERSScanController::buildBaseInitializationAction(double firstRegi
 	AMSequentialListAction3 *initializationAction = new AMSequentialListAction3(new AMSequentialListActionInfo3("VESPERS Scan Initialization Actions", "VESPERS Scan Initialization Actions"));
 	CLSSIS3820Scaler *scaler = VESPERSBeamline::vespers()->scaler();
 
-	AMListAction3 *stage1 = new AMListAction3(new AMListActionInfo3("VESPERS Initialization Stage 1", "VESPERS Initialization Stage 1"), AMListAction3::Sequential);
-	stage1->addSubAction(scaler->createContinuousEnableAction3(false));
-
-	AMListAction3 *stage2 = new AMListAction3(new AMListActionInfo3("VESPERS Initialization Stage 2", "VESPERS Initialization Stage 2"), AMListAction3::Sequential);
-
-	stage2->addSubAction(scaler->createStartAction3(false));
-	stage2->addSubAction(scaler->createScansPerBufferAction3(1));
-	stage2->addSubAction(scaler->createTotalScansAction3(1));
```

Comment:
We are also using those in the other beamlines. Won't it cause the same issue? 
And in SXRMB, the order of the actions are:
```
	scaler->createStartAction3(false);
	scaler->createContinuousEnableAction3(false);
	scaler->createScansPerBufferAction3(1);
	scaler->createTotalScansAction3(1);
```

Will this make the behavior different?